packages/expo-dev-menu/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unpublished
 
+[ios] Fix build failure on 0.76 with use frameworks ([#32341](https://github.com/expo/expo/pull/32341) by [@matinzd](https://github.com/matinzd))
```

Comment:
```suggestion
- [iOS] Fix build failure on 0.76 with use frameworks. ([#32341](https://github.com/expo/expo/pull/32341) by [@matinzd](https://github.com/matinzd))
```
It's also in the incorrect place. It should be in the `Bug Fixes` section.

Reply:
Done!